zerver/webhooks/linear/view.py

```diff
@@ -0,0 +1,167 @@
+from typing import Callable, Dict, Optional
+
+from django.http import HttpRequest, HttpResponse
+
+from zerver.decorator import webhook_view
+from zerver.lib.exceptions import UnsupportedWebhookEventTypeError
+from zerver.lib.request import REQ, has_request_variables
+from zerver.lib.response import json_success
+from zerver.lib.validator import WildValue, check_int, check_string, to_wild_value
+from zerver.lib.webhooks.common import check_send_webhook_message
+from zerver.models import UserProfile
+
+CONTENT_MESSAGE_TEMPLATE = "\n~~~ quote\n{message}\n~~~\n"
+
+ISSUE_CREATE_OR_UPDATE_TEMPLATE = (
+    "{type} [#{number} {title}]({url}) was {action} in team {team_name}"
+)
+
+ISSUE_REMOVE_TEMPLATE = "{type} **#{number} {title}** has been removed from team {team_name}."
+COMMENT_CREATE_OR_UPDATE_TEMPLATE = "{user} [{action}]({url}) on issue {issue_title}:"
+COMMENT_REMOVE_TEMPLATE = "{user} has removed a comment."
+
+
+def get_issue_created_or_updated_message(event: str, payload: WildValue, action: str) -> str:
+    message = ISSUE_CREATE_OR_UPDATE_TEMPLATE.format(
+        type="Issue" if event == "issue" else "Sub-Issue",
+        number=payload["data"]["number"].tame(check_int),
+        title=payload["data"]["title"].tame(check_string),
+        url=payload["url"].tame(check_string),
+        action=action,
+        team_name=payload["data"]["team"]["name"].tame(check_string),
+    )
+
+    has_description = "description" in payload["data"]
+    if has_description:
+        message += f":{CONTENT_MESSAGE_TEMPLATE.format(message=payload['data']['description'].tame(check_string))}"
+    else:
+        message += "."
+
+    to_add = []
+
+    priorityLabel = payload["data"]["priorityLabel"].tame(check_string)
+    if priorityLabel != "No priority":
+        to_add.append(f"Priority: {priorityLabel}")
+
+    has_assignee = "assignee" in payload["data"]
+    if has_assignee:
+        to_add.append(f"Assignee: {payload['data']['assignee']['name'].tame(check_string)}")
+
+    status = payload["data"]["state"]["name"].tame(check_string)
+    to_add.append(f"Status: {status}")
+
+    if to_add:
```

Comment:
Since we are unconditionally appending `status` to `to_add`, this condition will be always truthy.